src/Component.spec/context.spec.lua

```diff
@@ -92,12 +99,85 @@ return function()
 		assertDeepEqual(capturedContext, context)
 	end)
 
+	it("should not copy the context table if it doesn't need to", function()
+		local Parent = Component:extend("Parent")
+
+		function Parent:init()
+			self:__addContext("parent", "I'm here!")
+		end
+
+		function Parent:render()
+			-- Create some child element
+			return createElement(function() end)
+		end
+
+		local element = createElement(Parent)
+		local hostParent = nil
+		local hostKey = "Parent"
+		local parentNode = noopReconciler.mountVirtualNode(element, hostParent, hostKey)
+
+		local expectedContext = {
+			parent = "I'm here!",
+		}
+
+		assertDeepEqual(parentNode.context, expectedContext)
+
+		local childNode = oneChild(parentNode.children)
+
+		-- Parent and child should have the same context table
+		expect(parentNode.context, childNode.context)
```

Comment:
Do you mean to use `assert(a == b)` here?

Reply:
I sure did! Fortunately, it still passes.